<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="finito">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>51</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Maxbits">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxbits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>initrandom:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateRandoms">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>getrandom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ist">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ist</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="soll">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>soll:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kdatname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Keyfile:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savekey">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>save key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadkey">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>load key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="origfilelen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>original length of File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showtoprange">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>toprange:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="jumblekey">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Rejumble Key</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
